purescriptbook.code-workspace

```diff
@@ -0,0 +1,69 @@
+{
+  "folders": [
+    {
+      "path": "."
+    },
+    {
+      "path": "exercises/chapter2"
+    },
+    {
+      "path": "exercises/chapter3"
+    },
+    {
+      "path": "exercises/chapter4"
+    },
+    {
+      "path": "exercises/chapter5"
+    },
+    {
+      "path": "exercises/chapter6"
+    },
+    {
+      "path": "exercises/chapter7"
+    },
+    {
+      "path": "exercises/chapter8"
+    },
+    {
+      "path": "exercises/chapter9"
+    },
+    {
+      "path": "exercises/chapter10"
+    },
+    {
+      "path": "exercises/chapter11"
+    },
+    {
+      "path": "exercises/chapter12"
+    },
+    {
+      "path": "exercises/chapter13"
+    },
+    {
+      "path": "exercises/chapter14"
+    },
+    {
+      "path": "tmp/purcel"
+    },
+    {
+      "path": "tmp/purescript-comonad-ui-todos"
+    }
```

Comment:
Are these lines intended to be part of this PR?

Reply:
Npm my mistake, will send a fix, sorry